doc/life_of_a_language_feature.md

```diff
@@ -75,7 +104,43 @@ also coordinate the writing of additional tests.
 
 Implementated features will be released according to the implementation plan.
 The language team will contribute to:
+
   - Helping internal and external teams through any required migration.
+
```

Comment:
I'm curious why you added spaces between short bulleted list items. (It happened earlier in this doc, too.) I usually avoid the extra whitespace unless at least one of the items is a paragraph.

Reply:
This allows me to automatically wrap the bullets to 80 chars per line

Reply:
Note that this does unfortunately affect the final rendering.